packages/wrangler/src/delete.ts

```diff
@@ -98,3 +115,130 @@ async function deleteSiteNamespaceIfExisting(
 		logger.log(`🌀 Deleted asset namespace for Workers Site "${ns.title}"`);
 	}
 }
+
+type ServiceReference = {
+	name: string;
+	service: string;
+	environment: string;
+};
+
+type DurableObjectServiceReference = ServiceReference & {
+	durable_object_namespace_id: string;
+	durable_object_namespace_name: string;
+};
+
+type DispatchOutboundsServiceReference = ServiceReference & {
+	namespace: string;
+	params: { name: string }[];
+};
+
+export type ServiceReferenceResponse = {
+	services?: {
+		incoming: ServiceReference[];
+		outgoing: ServiceReference[];
+	};
+	durable_objects?: DurableObjectServiceReference[];
+	dispatch_outbounds?: DispatchOutboundsServiceReference[];
+};
+
+export type Tail = {
+	tag: string;
+	producer:
+		| {
+				service: string;
+				environment?: string;
+		  }
+		| { script: string };
+	consumer:
+		| {
+				service: string;
+				environment?: string;
+		  }
+		| { script: string };
+	log_push?: boolean;
+	config?: unknown;
+	created_on: string;
+	modified_on: string;
+};
+
+type ExtraConfirmResult = "not_needed" | "needed_denied" | "needed_confirmed";
```

Comment:
Is this intermediate string status needed? What about `true | false | undefined`, corresponding to the `force` parameter value?